pkg/controller/master/setting/handler.go

```diff
@@ -71,7 +71,7 @@ type Handler struct {
 	managedChartCache    ctlmgmtv3.ManagedChartCache
 	helmChartConfigs     ctlhelmv1.HelmChartConfigClient
 	helmChartConfigCache ctlhelmv1.HelmChartConfigCache
-	nodeClient           ctlcorev1.NodeClient
+	nodeClient           ctlcorev1.NodeController
```

Comment:
Please also check 
```
	skipHashCheckSettings = []string{
		settings.AutoRotateRKE2CertsSettingName,
	}
```

As this setting will be processed only once by default. If LH config was maunally/accidently changed later; Harvester will not sync again.

Add the related controller here, to have at least another chance, when the POD is restarted, to sync again.

Refer: https://github.com/harvester/harvester/pull/6396#issuecomment-2314476254

Reply:
Thanks for that. In this case I think we're probably OK because we pick up settings changes in `nodeOnChanged`, which happens more frequently.

Reply:
Ah, OK, I understand :-) I'll have to test that, but I suspect you're right and I need to add this setting to skipHashCheckSettings and/or document it.